proto/src/confstate.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::eraftpb::ConfState;
+
+fn eq_without_order(lhs: &[u64], rhs: &[u64]) -> bool {
+    for l in lhs {
+        if !rhs.contains(l) {
+            return false;
+        }
+    }
+    for r in rhs {
+        if !lhs.contains(r) {
+            return false;
+        }
+    }
+    true
+}
+
+// Returns a nil error if the inputs describe the same configuration.
+// On mismatch, returns a descriptive error showing the differences.
+pub fn conf_state_eq(lhs: &ConfState, rhs: &ConfState) -> bool {
```

Comment:
Seems the behavior and its comments are not matched

Reply:
Oh, my bad.